src/airhockey_vision/src/airhockey_vision/puck_trajectory.py

```diff
@@ -0,0 +1,63 @@
+import rospy
+import numpy as np
+from cv_bridge import CvBridge, CvBridgeError
+from collections import deque
+from std_msgs.msg import String
+from geometry_msgs.msg import PointStamped
+from std_msgs.msg import Header
+from airhockey_vision.msg import PuckState
+
+
+
+class TrajectoryCalculator:
+    def __init__(self):
+        self.buffer = deque(np.zeros((4,4)), maxlen = 5) # [x, y, x_dot, y_dot]
```

Comment:
would recommend putting these numbers as arugments (could have default values) in the constructor rather than as constants here, so we can do things like setting the buffer size or changing the state matrix size later on

Reply:
Also, why `np.zeros((4, 4))`? I think this might need to be `(5, 4)` because the deque has length of 5

Reply:
Actually come to think of it, can we initialize teh deque with an empty array? this would let us use the deque more as intended before it fills up (if we try to do a rolling average type thing)

Reply:
I've changed it so we're initializing it with an empty deque, no longer any np.zeros